ansible/files/envoy_config/lds.supabase.yaml

```diff
@@ -82,6 +82,25 @@ resources:
                                         name: ':path'
                                         string_match:
                                           contains: apikey=supabase_admin_key
+                        origin_protection_key_missing:
+                          permissions:
+                            - any: true
+                          principals:
+                            - not_id:
+                                header:
+                                  name: sb-opk
+                                  present_match: true
+                        origin_protection_key_not_valid:
+                          permissions:
+                            - any: true
+                          principals:
+                            - not_id:
+                                or_ids:
+                                  ids:
```

Comment:
Any reason for not doing a direct comparison here rather than an array check?

Reply:
There can be multiple keys here, coming from the `trusted_keys` column, to enable easy rotation.